SimTracker/TrackerMaterialAnalysis/test/trackingMaterialProducer10GeVNeutrino_ForPhaseII.py

```diff
@@ -52,7 +52,7 @@
    from Configuration.AlCa.GlobalTag import GlobalTag
    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
 else:
-   process.load('Configuration.Geometry.GeometryExtended2026D49_cff')
+   process.load('Configuration.Geometry.GeometryExtended2026D49Reco_cff')
```

Comment:
Why this requires the reco geometry?

Reply:
The reason is HCAL went through a phase where HE was considered a possibility for Phase2 detector. That needs the grouping at SIM and RECO level to be different. Later Reco regrouping was needed in some SIM related studies. That is why it is always better to include the so called RECO geometry which any way has the SIM geometry inside it.